.gitignore

```diff
@@ -31,3 +31,5 @@ build/
 
 ### VS Code ###
 .vscode/
+
+.run/
```

Comment:
what do you mean? xd

Reply:
> what do you mean? xd

I mean i seriously need these files, why should they be excluded.

Reply:
then revert, but why do i have to change it everytime I want to run the BE?

Reply:
You just have to name the module accordingly to the name in the xml file.